tests/containers/accountLanding/AccountLandingContainer-test.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * AccountLandingContainer-test.jsx
+ * Created by Lizzie Salita 2/7/18
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+
+import * as FiscalYearHelper from 'helpers/fiscalYearHelper';
+import AccountLandingContainer from 'containers/accountLanding/AccountLandingContainer';
+
+import { mockData, mockParsed } from './mockFederalAccounts';
+
+jest.mock('helpers/accountLandingHelper', () => require('./accountLandingHelper'));
+
+// mock the child component by replacing it with a function that returns a null element
+jest.mock('components/accountLanding/AccountLandingContent', () =>
+    jest.fn(() => null));
+
+describe('AccountLandingContainer', () => {
+    it('should make an API request on mount', async () => {
+        // mount the container
+        const container = mount(<AccountLandingContainer />);
+        const parseAccounts = jest.fn();
+        container.instance().parseAccounts = parseAccounts;
+
+        await container.instance().accountsRequest.promise;
+
+        expect(parseAccounts).toHaveBeenCalledTimes(1);
+    });
+
+    describe('showColumns', () => {
+        it('should build the table', async () => {
+            // mount the container
+            const container = mount(<AccountLandingContainer />);
+
+            await container.instance().accountsRequest.promise;
+
+            // validate the state contains the correctly parsed values
+            const fy = FiscalYearHelper.defaultFiscalYear();
+            const expectedState = [
+                {
+                    columnName: "account_number",
+                    defaultDirection: "desc",
+                    displayName: "Account Number"
+                },
+                {
+                    columnName: "account_name",
+                    defaultDirection: "asc",
+                    displayName: "Account Name"
+                },
+                {
+                    columnName: "managing_agency",
+                    defaultDirection: "asc",
+                    displayName: "Managing Agency"
+                },
+                {
+                    columnName: "budgetary_resources",
+                    defaultDirection: "desc",
+                    displayName: `${fy} Budgetary Resources`
+                }
+            ];
+
+            expect(container.state().columns).toEqual(expectedState);
+        });
+    });
+
+    describe('parseAccounts', () => {
+        it('should parse the API response and update the container state', () => {
+            const container = mount(<AccountLandingContainer />);
```

Comment:
Where we are not testing the component lifecycle, we should use `shallow` instead in order to explicitly call (only) the method we are testing.